### PR TITLE
Add SMIT Gallery Patients

### PR DESCRIPTION
**Pull request by ivan-c**
- Add Patient resource stubs for mocked patients
- uploaded via `curl -H "Content-Type: application/json" --data @SMIT-gallery-patients.json https://r4.smarthealthit.org/`

### CODE REVIEW

**Review by pbugni — APPROVED** [review]
Looks solid @ivan-c 
I don't know about their willingness to accept changes - would it be wise to work in labs and contract data for these patients before requesting a merge from SMIT?

**Comment by mcjustin** [comment]
@pbugni  wrote
> Looks solid @ivan-c I don't know about their willingness to accept changes - would it be wise to work in labs and contract data for these patients before requesting a merge from SMIT?

That's what I was wondering too... along those lines, does our mock data endpoint have anything other than PDMP data?

**Comment by ivan-c** [comment]
>would it be wise to work in labs and contract data for these patients before requesting a merge from SMIT?

@pbugni 
Good point. I'll plan on merging this, but won't submit a PR to the [SMIT repo](https://github.com/smart-on-fhir/generated-sample-data) until we have those

> does our mock data endpoint have anything other than PDMP data?

@mcjustin 
Nope, only RxHistory/MedicationDispensed XML; our PDMP facade converts those into FHIR MedicationRequest

**Pull request merged**
